utils/openshift_acme.py

```diff
@@ -1,76 +1,38 @@
 ACME_DEPLOYMENT = """
-apiVersion: extensions/v1beta1
 kind: Deployment
+apiVersion: apps/v1
 metadata:
   name: %(deployment_name)s
   labels:
     app: openshift-acme
 spec:
-  progressDeadlineSeconds: 600
   replicas: 1
```

Comment:
Interestingly enough, the templates from the doc set this to 2 whereas in the past it was set to 1...